recipes-demos/ti-lvgl-demo/ti-lvgl-demo.bb

```
@@ -0,0 +1,49 @@
+DESCRIPTION = "High Resolution OOB Demo"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
```

I don't see a LICENSE file in `gitsm://github.com/texasinstruments/ti-lvgl-demo.git`
Where are we getting the above LICENSE from?

---

@jsuhaas22 Handle this as well
```suggestion
LIC_FILES_CHKSUM = "file://${WORKDIR}/git/LICENSE;md5=<md5sum-value>"
```